### DISCUSSION TITLE
Generate Release Notes not working

### DISCUSSION THREAD

**Discussion by aakashbajaj** [community/community#120720] [Repositories]
### Select Topic Area

Bug

### Body

Generate Release Notes button is not working anymore.

It tries to generate diff from the same tag version.

<img width="882" alt="image" src="https://github.com/community/community/assets/18542246/01ddefea-cece-43a5-b533-d7e00a19ad23">

I believe the issue is with the fetch call being made, which doesn't include the previous tag version.
<img width="850" alt="image" src="https://github.com/community/community/assets/18542246/e16edbad-7fb1-4abc-bc5a-61a959c69b81">

**Comment by david-wei-01001** [comment]
It looks like the issue you're facing with the "Generate Release Notes" button is due to the fact that the comparison is being made between the same tag version, as you've pointed out. Here's what could be happening and what you might consider doing to resolve the issue:

1. **Incorrect Previous Tag**: The fetch call for release notes should compare the current tag with the previous one. If the `previous_tag_name` parameter is missing or incorrect, it will not generate the correct release notes. This could be a bug or a misconfiguration in the release draft.

2. **API Call**: The screenshot of the fetch call shows that it's trying to generate release notes comparing the same version (`v2.49.0...v2.49.0`), which would indeed not yield any result because there are no differences within the same tag.

3. **Manual Generation**: If automatic generation is not working, you may need to manually generate the release notes. You can do this by manually comparing the tags on GitHub and listing out the pull requests and commits that are part of the new release.

4. **GitHub Issue**: If this functionality was working before and has suddenly stopped, it's possible there's a temporary issue with GitHub. You can check [GitHub's status page](https://www.githubstatus.com/) to see if there are any operational issues that could be affecting this feature.

5. **Report the Issue**: If you believe this is a bug with GitHub, you should report it to GitHub Support. Provide them with all the relevant details and screenshots to help them understand the issue.

6. **Check for Updates**: If you're using GitHub Desktop or any third-party tools for release management, ensure they're updated to the latest version, as this might be an issue that has been fixed in a newer release.

7. **Use GitHub API**: As a workaround, you could use the GitHub API to generate release notes. Write a script to fetch commits between tags and format them as release notes.

As a direct next step, try generating the release notes by specifying the range of tags manually, ensuring that you're not comparing the same tag with itself. If the issue persists or you're unsure how to proceed, reaching out to GitHub Support would be advisable.

**Reply by aakashbajaj** [reply]
Where should the issue be reported ?

**Comment by queenofcorgis** [comment]
@aakashbajaj you're in the right place. You can report in the Community and ensure the `bug` label is on the Discussion (which is taken care of). 

Please note: This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Reply by PrzemyslawKlys** [reply]
I noticed exactly the same problem for few days. The version on the left side is always same or higher then on the right hand side.
